mim/utils/default.py

```diff
@@ -32,4 +32,5 @@
 if not os.path.exists(DEFAULT_CACHE_DIR):
     os.makedirs(DEFAULT_CACHE_DIR)
 
-MMPACKAGE_PATH = os.path.join(site.getsitepackages()[0], 'mmpackage.txt')
+MMPACKAGE_PATH = os.path.join(
```

Comment:
May indicate why we do not use site.getsitepackage() in the comments.

Reply:
It is an issue with virtualenv.
refer to https://github.com/pypa/pipenv/issues/3824